frontend/src/components/datasets/ActionsMenu.tsx

```diff
@@ -32,7 +38,7 @@ export const ActionsMenu = (props: ActionsMenuProps): JSX.Element => {
 			<Button
 				sx={{ minWidth: "auto" }}
 				variant="contained"
-				href={`${config.hostname}/api/v2/datasets/${datasetId}/download`}
+				onClick={() => download(datasetId)}
```

Comment:
@lmarini I recall we intentionally changed from the download function to just embed href to handle large file download at a point: https://github.com/clowder-framework/clowder2/pull/692
Do we have to change it back?